src/boot/ksdk1.1.0/devADXL362.h

```diff
@@ -35,6 +35,9 @@
 	POSSIBILITY OF SUCH DAMAGE.
 */
 
+#ifndef warp_devADXL362_ENABLE
```

Comment:
Please see the coding conventions at https://github.com/physical-computation/Warp-firmware/blob/master/doc/README-CodingConventions.md